### PR TITLE
UI: Collapsible sidebar and fixes

### PR DESCRIPTION
**Pull request by Amogh-Bharadwaj**
- Collapsible sidebar
- Remove extra borders from tables
- Change dropdown colors

https://github.com/PeerDB-io/peerdb/assets/65964360/e5c1c47f-c4b4-431f-ab60-a9881330dd5a

Screenshot 2024-02-29 at 10 42 05 PM

### CODE REVIEW

File: ui/components/SidebarComponent.tsx [hunk 1]
```diff
@@ -73,40 +104,46 @@ export default function SidebarComponent(props: {}) {
...
+            <div style={centerFlexStyle}>
+              <Label as='label' style={{ textAlign: 'center', fontSize: 15 }}>
+                {' '}
+                <b>Version: </b>
+                {isLoading ? 'Loading...' : version?.version}
+              </Label>
+            </div>
+          ) : (
+            <></>
+          )
```
Comment (serprex) [inline ui/components/SidebarComponent.tsx:124]:
```suggestion
          sidebarState == 'open' && (
            <div style={centerFlexStyle}>
              <Label as='label' style={{ textAlign: 'center', fontSize: 15 }}>
                {' '}
                <b>Version: </b>
                {isLoading ? 'Loading...' : version?.version}
              </Label>
            </div>
          )
```

File: ui/components/SidebarComponent.tsx [hunk 2]
```diff
@@ -73,40 +104,46 @@ export default function SidebarComponent(props: {}) {
...
         }
       >
         <SidebarItem
           as={Link}
           href={'/peers'}
           leadingIcon={<Icon name='cable' />}
         >
-          Peers
+          {sidebarState == 'open' && 'Peers'}
```
Comment (serprex) [inline ui/components/SidebarComponent.tsx:132]:
```suggestion
          {sidebarState === 'open' && 'Peers'}
```

File: ui/components/SidebarComponent.tsx [hunk 3]
```diff
@@ -21,33 +22,63 @@ const centerFlexStyle = {
...
+          >
+            {sidebarState == 'closed' ? (
+              <></>
+            ) : (
+              <Label as={Link} href='/'>
+                <div className='cursor-pointer'>
+                  <BrandLogo />
+                </div>
+              </Label>
+            )}
```
Comment (serprex) [inline ui/components/SidebarComponent.tsx:62]:
```suggestion
            {sidebarState === 'open' && (
              <Label as={Link} href='/'>
                <div className='cursor-pointer'>
                  <BrandLogo />
                </div>
              </Label>
            )}
```

**Comment by serprex** [comment]
nits & CI aside, lgtm

File: ui/components/SidebarComponent.tsx (hunk 3 above)
Comment (Amogh-Bharadwaj) [reply ui/components/SidebarComponent.tsx]: Sidebar component's attributes don't accept `boolean | JSXElement` which is the resulting type of condition && element

File: ui/components/SidebarComponent.tsx (hunk 1 above)
Comment (Amogh-Bharadwaj) [reply ui/components/SidebarComponent.tsx]: Sidebar component's attributes don't accept `boolean | JSXElement` which is the resulting type of condition && element

**Review by iskakaushik — APPROVED** [review]

**Pull request merged**